manyfews/calculations/models.py

```diff
@@ -37,3 +37,18 @@ class InitialCondition(models.Model):
     storage_level = models.FloatField()
     slow_flow_rate = models.FloatField()
     fast_flow_rate = models.FloatField()
+
+
+class RainAndEvapotranspiration(models.Model):
+    date = models.DateTimeField()
+    location = models.PointField(default=Point(0, 0))
+    forecast_hour = models.IntegerField(default=int(0))
+    rain_fall = models.FloatField()
+    potential_evapotranspiration = models.FloatField()
+
+
+class PotentialRiverFlows(models.Model):
```

Comment:
Again, could you add a comment/docstring explaining that this is the output from the river flows model?

Reply:
Do I still need to add comments to the two tables?